index.html

```diff
@@ -13,20 +13,20 @@ <h2>ECIP Types</h2>
 
 <p>ECIPs are separated into a number of types, and each has its own list of ECIPs.</p>
 
-<h3>Standard Track ({{site.specs|where:"type","Standards Track"|size}})</h3>
+<h3>Standard Track ({{site.specs|where:"lang","en"|where:"type","Standards Track"|size}})</h3>
```

Comment:
Cool stuff. This will make it easier to add more languages in the future for all of them and explore each ECIP.